### PR TITLE
Upgrade docker dependencies

### PR DESCRIPTION
**Pull request by PierreBesson**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

Upgrades to docker dependencies including the new JHipster Console (v2.1.0).

### CODE REVIEW

File: generators/generator-constants.js [hunk 1]
```diff
@@ -31,15 +31,16 @@ const DOCKER_ELASTICSEARCH = 'elasticsearch:2.4.1';
...
-const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v2.0.1';
-const DOCKER_CONSUL = 'consul:0.7.2';
-const DOCKER_CONSUL_CONFIG_LOADER = 'jhipster/consul-config-loader:v0.2.1';
-const DOCKER_PROMETHEUS = 'prom/prometheus:v1.4.1';
-const DOCKER_PROMETHEUS_ALERTMANAGER = 'prom/alertmanager:v0.5.1';
-const DOCKER_GRAFANA = 'grafana/grafana:4.0.2';
+const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v2.1.0';
+const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v2.1.0';
+const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v2.1.0';
+const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'pbesson/jhipster-import-dashboards:latest';
```
Comment (PierreBesson) [inline generators/generator-constants.js]: @pascalgrimaud This is temporary until the following docker image is created: `jhipster/jhipster-import-dashboards`. Can you create the automated on docker hub plz.

**Comment by pascalgrimaud** [comment]
@PierreBesson : I will review it probably tonight or tomorrow

File: generators/generator-constants.js (hunk 1 above)
Comment (pascalgrimaud) [inline generators/generator-constants.js]: Just done with my phone lol :D :D
Can you look at https://hub.docker.com/r/jhipster/jhipster-import-dashboards/tags/ ?

**Comment by PierreBesson** [comment]
I have everything working now but plz wait before merging as I need to add a README file that explain that the import-dashboard container must be restarted after creating the index-pattern.

**Comment by pascalgrimaud** [comment]
LGTM (only reading your code, I can't download all Docker images right now to really test)
As you said, we need to complete the documentation

**Comment by PierreBesson** [comment]
@pascalgrimaud I'm done. You can do a final review and merge.

**Comment by jdubois** [comment]
I'm getting errors:

    Error importing directory /tmp/: Failed to import index-pattern: Failed to load directory /tmp/index-pattern:
  error loading /tmp/index-pattern/logstash.json: couldn't load json. Error: Put http://jhipster-
    elasticsearch:9200/.kibana/index-pattern/logstash-*: dial tcp 172.18.0.3:9200: getsockopt: connection refused

As I'm going to release `4.5.5` I might revert this commit, otherwise I'm stucked

**Comment by jdubois** [comment]
To be more precise: the "jhipster-import-dashboards" image fails with this warning

**Comment by jdubois** [comment]
Sorry guys but I need to revert https://github.com/jhipster/generator-jhipster/pull/5882/commits/18eedf624464844012f23ad5955c4154545ffcaf -> this is what makes everything fails, and reverting it makes Kibana works again. I need to do an emergency release, this will be for our next release

**Comment by PierreBesson** [comment]
@jdubois I think this is normal as Elasticsearch is not yet up so the dashboards can't get imported into it. I'm trying to solve this using docker compose healthchecks but meanwhile you can simply restart the import-dashboard container as I have written in the readme.

The new console is based on the official docker images from elastic since they deprecated the docker hub images.

**Comment by jdubois** [comment]
I'm sorry but:

- This needs to work out of the box, that's the "JHipster promise"
- I didn't read your doc (sorry), but I did restart the image, it didn't work any better

**Comment by jdubois** [comment]
I really need to do an emergency release now, this can wait for the next minor release

**Comment by pascalgrimaud** [comment]
Yes, it can be reviewed later

**Comment by PierreBesson** [comment]
It's OK @jdubois. I think it doesn't work out of the box due to some changes in the next version that prevent the index pattern from being created when there is still no logs received. I will try to come up with a solution.

**Pull request merged**